lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -522,6 +522,51 @@ void TypeLoweringVisitor::visitDecl(MemOp op) {
 
   opsToRemove.push_back(op);
 }
+void static filterAnnotations(ArrayAttr annotations,
```

Comment:
`static void`.  Also please separate from prev function with a new line and add a block comment.